packages/core/content-releases/server/src/controllers/release-action.ts

```diff
@@ -29,6 +30,51 @@ const releaseActionController = {
     };
   },
 
+  async createMany(ctx: Koa.Context) {
+    const releaseId: CreateManyReleaseActions.Request['params']['releaseId'] = ctx.params.releaseId;
+    const releaseActionsArgs: CreateManyReleaseActions.Request['body'] = ctx.request.body;
+
+    await Promise.all(
+      releaseActionsArgs.map((releaseActionArgs) => validateReleaseAction(releaseActionArgs))
+    );
+
+    const releaseService = getService('release', { strapi });
+
+    const releaseActions = await strapi.db.transaction(async () => {
+      const releaseActions = await Promise.all(
+        releaseActionsArgs.map(async (releaseActionArgs) => {
+          try {
+            const action = await releaseService.createAction(releaseId, releaseActionArgs);
+
+            return action;
+          } catch (error) {
+            if (
+              error instanceof errors.ValidationError &&
+              error.message ===
+                `Entry with id ${releaseActionArgs.entry.id} and contentType ${releaseActionArgs.entry.contentType} already exists in release with id ${releaseId}`
+            ) {
+              return null;
+            }
+
+            throw error;
```

Comment:
does this unhandled error result in a 500 status error or a 400?

Reply:
ah my bad , completey missed the goal of this bit of code.
Would it be possible to throw a custom Error instead of checking the exact message string? If someone changes the error string this might be unnoticed.